prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html

```diff
@@ -80,6 +80,9 @@
             <mat-error *ngIf="licenseNumber.hasError('alphanumeric')">
               Must contain only alphanumeric characters
             </mat-error>
+            <mat-error *ngIf="licenseNumber.hasError('numeric') || licenseNumber.hasError('length')">
+              Must contain only 5 numeric characters
+            </mat-error>
```

Comment:
Maybe it's also worth putting a `maxLength="5"` for the input on line 76 of this file.

Reply:
Short answer no.

This input is used for many different license types and since this restriction is only for a certain college we don't want to hard code the field. Also we could then store it as a variable, but then we would be tracking validators in two places and it is not needed.

Reply:
Yep was just looking at it, commented too soon.